playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -31,6 +31,65 @@ class OomKillerEnricherParams(ActionParams):
 NODE_MEMORY_THRESHOLD = 0.95
 
 
+@action
+def pod_oom_killer_enricher(
+    event: PodEvent,
+):
+    pod = event.get_pod()
```

Comment:
Just to be on the safe side, I would add a check if the pod object exists.
This action requires PodEvent.
Alert is a pod event, but might not have a pod object for some alerts

Reply:
Got it,
Is it possible to have an OOMKill event without a pod object? 
if so then i need to change this enricher to support that.